README.md

```diff
@@ -0,0 +1,34 @@
+# Brainkeeper
+
+[![Build Status](https://drone.dev.sart.solutions/api/badges/brainkeeper/frontend/status.svg)](https://drone.dev.sart.solutions/brainkeeper/frontend)
+[![codecov](https://codecov.io/gh/brainkeeper/frontend/branch/master/graph/badge.svg)](https://codecov.io/gh/brainkeeper/frontend)
+[![Docker Pulls](https://img.shields.io/docker/pulls/hmbrainkeeper/frontend.svg)](https://hub.docker.com/r/hmbrainkeeper/frontend/)
+[![License: MIT](https://img.shields.io/badge/License-MIT-brightgreen.svg)](https://opensource.org/licenses/MIT)
+
+[**Visit the latest stable version**](https://app.brainkeeper.sart.solutions/)
+
+---
+
+### Introduction
+This Project was started by a [group of students](https://github.com/orgs/brainkeeper/people) from Munich University of Applied Sciences together with the Digital Transformation Lab and the local Alzheimer's Association
+
+### Overview
+Brainkeeper is a small specialized memory training app, which aims to improve, or slow the degeneration of the ability to remember and recognize faces and names.
+
+### Problem description
+A big problem of people with high age or Alzheimer’s often is, that they don’t recognize people like family members, old friends or medical personal they interact with on a regular basis. They often are ashamed because of that, and as a result of that stop socializing.
+
+### Idea
+Our Idea was to build a simple to use App, usable on all devices, where the Player can train there brain with faces of people who are of interest to him. 
+
+### Result
+We designed this simplistic App where the focus of the start screen is to just start playing. 
+When the user wants to see his progress, he can do so after clicking the “Statistics” button, where we show present him all the persons that he has already recognized, and how often he has done so, but only that. We intentionally only give positive statistics and feedback, so that we don’t discouraged our player, when his menta capabilities may be declining, because stopping mental training may only accelerate the decay. 
```

Comment:
he can do so by clicking... where we display all the persons...
...so that the player is not discouraged by an eventual decline of their mental capabilities, as stopping training completely will merely accelerate the decline.